contracts/WildcardsQV.sol

```diff
@@ -223,6 +226,30 @@ contract WildcardsQV is Initializable {
         );
     }
 
+    function safeFundsTransfer(address payable recipient, uint256 amount)
+        internal
+    {
+        // attempt to send the funds to the recipient
+        (bool success, ) = recipient.call.value(amount)("2300");
+        // if it failed, update their credit balance so they can pull it later
+        if (success == false) {
+            failedTransferCredits[recipient] += amount;
+        }
+    }
```

Comment:
OKay this makes sense. So basically in case the address can't receive the payout, they get credits, from which they can withdraw later.